src/main/java/org/opensrp/queue/QueueHelper.java

```diff
@@ -0,0 +1,126 @@
+package org.opensrp.queue;
+
+import com.ibm.fhir.model.format.Format;
+import com.ibm.fhir.model.parser.FHIRParser;
+import com.ibm.fhir.model.parser.exception.FHIRParserException;
+import com.ibm.fhir.model.resource.DomainResource;
+import com.ibm.fhir.model.resource.QuestionnaireResponse;
+import org.opensrp.repository.ClientsRepository;
+import org.opensrp.repository.EventsRepository;
+import org.opensrp.repository.LocationRepository;
+import org.opensrp.repository.TaskRepository;
+import org.opensrp.service.PlanService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.smartregister.domain.Action;
+import org.smartregister.domain.Jurisdiction;
+import org.smartregister.domain.PlanDefinition;
+import org.smartregister.pathevaluator.PathEvaluatorLibrary;
+import org.smartregister.pathevaluator.TriggerType;
+import org.smartregister.pathevaluator.dao.QueuingHelper;
+import org.smartregister.pathevaluator.plan.PlanEvaluator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+@Component
+@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
```

Comment:
Why is this used, this will have an impact on memory allocation since a bean will created for each task

Reply:
Removed.